### PR TITLE
Fix for missing text in Notifications drawer

### PR DESCRIPTION
**Pull request by akhilkr128**
Before
------------

Screenshot 2023-05-08 at 2 19 00 PM

After
------------
Screenshot 2023-05-08 at 10 21 54 AM

Additional Screenshots here - https://github.com/ManageIQ/manageiq-ui-service/pull/1848#issuecomment-1539880408

### CODE REVIEW

**Comment by jeffibm** [comment]
Adding few screenshots for future reference

**Collapsed**
image
image
image
image

**Expanded**
image
image
image
image

**Review by jeffibm — APPROVED** [review]

File: client/app/core/navigation/navigation-controller.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ export function NavigationController (Text, Navigation, Session, API_BASE, Shopp
       helpMenu: RBAC.hasAny(['about', 'product', 'documentation'])
     }
     EventNotifications.setToastDisplay(vm.permissions.suiNotifications)
+    const appBasePath = process.env.NODE_ENV === 'production' ? '' : 'assets/'
```
Comment (Fryguy) [inline client/app/core/navigation/navigation-controller.js:63]: How have you verified this in production?

Comment (jeffibm) [reply client/app/core/navigation/navigation-controller.js:63]: we tested locally by running `yarn build`

**Comment by Fryguy** [comment]
Backported to `petrosian` in commit 3fa7f62c3af98255dfffd167ab91d2724b0f50f1.

```text
commit 3fa7f62c3af98255dfffd167ab91d2724b0f50f1
Author: Jason Frey <fryguy9@gmail.com>
Date:   Wed May 10 14:14:37 2023 -0400

    Merge pull request #1848 from akhilkr128/notification_bug
    
    Fix for missing text in Notifications drawer
    
    (cherry picked from commit e6abef09cec46acce920f1318607013e7bde666f)
```

**Comment by jeffibm** [comment]
Just linking the PR here - https://github.com/ManageIQ/manageiq-ui-service/pull/1849

**Pull request merged**